FND/Arrangements/Assessments.rdf

```diff
@@ -138,16 +184,36 @@
 	
 	<owl:Class rdf:about="&fibo-fnd-arr-asmt;ValueAssessment">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-asmt;AssessmentEvent"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-dt-oc;hasOutput"/>
+				<owl:onClass rdf:resource="&fibo-fnd-arr-asmt;Appraisal"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-asmt;hasAppraiser"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-pty-rl;AgentInRole"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
 		<rdfs:label>value assessment</rdfs:label>
 		<skos:definition>assessment event to estimate the value of something</skos:definition>
 	</owl:Class>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-arr-asmt;hasAppraiser">
-		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-rl;hasAgentInRole"/>
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-arr-asmt;Appraiser"/>
```

Comment:
No this is wrong: Appraiser is a Class not a Property!

Reply:
Fixed.